### ISSUE TITLE
Loss does not converge

### ISSUE THREAD

**Issue by Eaphan** [Vegeta2020/SE-SSD#72]
I use the default config and take the CIA-SSD as pretrained model. But I still can't reproduce the performance. To be specific, the loss doesn't converge. The total loss starts from 0.5, increases to 0.6x  and finnaly ends at about 0.55. Do the change meet our expectations?
@Vegeta2020 Thanks for your attention.

Besides, there are some warnings in my training logs:

> Warning:  using Python fallback for SyncBatchNorm, possibly because apex was installed without --cuda_ext.  The exception raised when attempting to import the cuda backend was:  No module named 'syncbn'

I'm not sure whether the syncbn of Python fallback works.

**Comment by Vegeta2020** [comment]
"syncbn" seems the issue of apex, you may have to install it well. The loss also doesn't reach the expected trend.
